content/code-signing/android-code-signing.md

```diff
@@ -25,7 +25,7 @@ As a keystore can hold multiple keys, each key in it must have an **alias**. Bot
 {{<notebox>}}
 
 - Please note that every app must be signed using the same key throughout its lifespan.
-- If you're building Android App Bundles, you additionally need to [enroll your app into app signing by Google Play](https://support.google.com/googleplay/android-developer/answer/7384423).
+- If you're building Android App Bundles, you additionally need to [enrol your app into app signing by Google Play](https://support.google.com/googleplay/android-developer/answer/7384423).
```

Comment:
Enroll is American English and enroll is British English, which do we follow?
https://www.grammar.com/enrol_vs._enroll